docs/getting_started/gce.rst

```diff
@@ -44,13 +43,13 @@ customization. In the next sections, we'll explain how to obtain these settings.
 Basic Settings
 ^^^^^^^^^^^^^^
 
-``project``, ``region`` and ``zones`` are unique values for each project in Google Compute
+``Project``, ``region`` and ``zones`` are unique values for each project in Google Compute
```

Comment:
Agreed

From: Langston Barrett [mailto:notifications@github.com]
Sent: Tuesday, July 05, 2016 1:12 PM
To: CiscoCloud/mantl <mantl@noreply.github.com>
Cc: John Scott Wilson -T (johnswil - RANDSTAD NORTH AMERICA LP at Cisco) <johnswil@cisco.com>; Mention <mention@noreply.github.com>
Subject: Re: [CiscoCloud/mantl] Copy edits to docs/getting_started/gce (#1641)


In docs/getting_started/gce.rst<https://github.com/CiscoCloud/mantl/pull/1641#discussion_r69629048>:

> @@ -44,13 +43,13 @@ customization. In the next sections, we'll explain how to obtain these settings.

>  Basic Settings

>  ^^^^^^^^^^^^^^

>

> -``project``, ``region`` and ``zones`` are unique values for each project in Google Compute

> +``Project``, ``region`` and ``zones`` are unique values for each project in Google Compute

This shouldn't be capitalized, it is a configuration value.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/CiscoCloud/mantl/pull/1641/files/bbb31025fdbd79bb5cbdb1600d7af0b76d55cd03#r69629048>, or mute the thread<https://github.com/notifications/unsubscribe/AR2-fJ7RCW6YhTtmNFnU4c4eU3pTU8wdks5qSrqGgaJpZM4JEsRy>.